### PR TITLE
Fixes #568 Fixes singularity install failures

### LINKED ISSUE #568: cli: singularity installation on travis fails sometimes
we're experiencing an issue on Travis that is causing tests to fail sporadically. Looks like the problem is related to adding a key for a repository. One workaround is to download the [`.deb` file](http://neurodeb.pirsquared.org/pool/main/s/singularity-container/singularity-container_2.6.1-2~nd16.04+1_amd64.deb) directly using curl/wget and then run `apt install ./<file>.deb`.

### PR DESCRIPTION
**Pull request by JayjeetAtGithub**
Fixes unpredictable singularity install failures

### CODE REVIEW

**Comment by JayjeetAtGithub** [comment]
@ivotron The build ran successfully two times consecutively and it passed now ! Please have a look! Thanks.

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ python:
 - "3.7"
 services: docker
 before_install:
-- sudo wget -O- http://neuro.debian.net/lists/xenial.us-ca.full | sudo tee /etc/apt/sources.list.d/neurodebian.sources.list
-- sudo apt-key adv --recv-keys --keyserver hkp://pool.sks-keyservers.net:80 0xA5D32F012649A5A9 
 - sudo apt-get update
-- sudo apt-get install -y singularity-container
+- sudo apt-get install neurodebian-popularity-contest
```
Comment (ivotron) [inline .travis.yml]: This seems to have slipped in; shouldn't be needed.

Comment (JayjeetAtGithub) [reply .travis.yml]: This is required as otherwise i was getting dependency error. `unment dependency : neurodebian-popularity-contest`

Comment (ivotron) [reply .travis.yml]: that might be solved with `apt-get -f install` after doing `apt install`

**Comment by JayjeetAtGithub** [comment]
@ivotron  I have updated the PR ! Please have a look. Thanks.

**Pull request merged**